templates/newrelic-stack-set.yml

```diff
@@ -60,13 +64,14 @@ Resources:
         Statement:
           - Effect: Allow
             Principal:
-              AWS: 'arn:aws:iam::754728514883:root'
+              AWS: !Sub 'arn:aws:iam::${NewRelicTrustedPrincipal}:root'
```

Comment:
same as above.

Reply:
since NewRelic AWS account may use different role to perform STS, we need to assign it to root to allow IAM users/roles with proper external ID to assume it.